### PR TITLE
git round trip working... sorta

### PR DESCRIPTION
**Pull request by kzantow**

### CODE REVIEW

File: src/main/js/SaveApi.js [hunk 1]
```diff
@@ -6,22 +6,23 @@ import { Fetch, getRestUrl, sseService, loadingIndicator, capabilityAugmenter }
 export class SaveApi {
 
     indexRepo(organization, teamName, repoName) {
-        const createUrl = `${getRestUrl({organization})}/pipelines/`;
-
+        const createUrl = repoName ? `${getRestUrl({organization})}pipelines/` : `${getRestUrl({organization, pipeline: teamName})}`;
+        const repo = repoName ? repoName : teamName; // may not be an org folder
+        const jenkinsClass = repoName ? 'io.jenkins.blueocean.blueocean_github_pipeline.GithubPipelineCreateRequest'
+                : 'io.jenkins.blueocean.blueocean_git_pipeline.GitPipelineUpdateRequest';
```
Comment (i386) [inline src/main/js/SaveApi.js]: @kzantow where is `GitPipelineUpdateRequest` defined?

Comment (kzantow) [inline src/main/js/SaveApi.js]: @i386 in https://github.com/jenkinsci/blueocean-plugin/pull/931

**Comment by michaelneale** [comment]
I tried this with ye old trusty https://github.com/multibranchorg/app-store-demo - I saw it save back and seemed got with ssh and "system" (which is what I would expect). 

I also tried it with https with a personal token for github. 

Both worked, neither seemed slower on that repo than with github, maybe load was a tiny bit longer, but not really.. 

WHAT WITCHCRAFT IS THIS! 

nice one.

**Comment by kzantow** [comment]
Superseded by: https://github.com/jenkinsci/blueocean-pipeline-editor-plugin/pull/48

**Pull request closed without merging**